hasher-matcher-actioner/terraform/webapp/outputs.tf

```diff
@@ -7,3 +7,7 @@ output "s3_bucket_name" {
 output "cloudfront_distribution_domain_name" {
   value = var.include_cloudfront_distribution ? aws_cloudfront_distribution.webapp[0].domain_name : "no-cloudfront-distribution"
 }
+
+output "ui_url" {
+  value = aws_s3_bucket.webapp.website_endpoint
```

Comment:
did you test this`include_cloudfront_distribution` set to false? I think this might error.

If you look at the check above you can have this return empty string if set to false.

Reply:
Talked with Barrett offline. Confirmed this works with `include_cloudfront_distribution` set to true/false and if a shared user pool is used